node-runner-cli/Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-radixdlt_core_client = {git = "https://github.com/radixdlt/python-core-client", editable = true, ref = "1.0-beta.1"}
+core_client = {git = "https://github.com/radixdlt/python-core-client", editable = true}
```

Comment:
Can we specify an _exact_ tag ref for this? 👍  - to avoid other people or the build pipeline building with a cached / incorrect version?

Reply:
right